src/main/java/org/kairosdb/core/opentracing/HttpHeadersCarrier.java

```diff
@@ -0,0 +1,33 @@
+package org.kairosdb.core.opentracing;
+
+import io.opentracing.contrib.web.servlet.filter.HttpServletRequestExtractAdapter;
+import io.opentracing.propagation.TextMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+import java.util.Iterator;
+import java.util.Map;
+
+public class HttpHeadersCarrier implements TextMap{
```

Comment:
Please run "format the code" with IDEA for this file. For instance, there should be a space between `TextMap` and `{`